tmk_core/protocol/host.c

```diff
@@ -73,14 +97,7 @@ led_t host_keyboard_led_state(void) {
 
 /* send report */
 void host_keyboard_send(report_keyboard_t *report) {
-#ifdef BLUETOOTH_ENABLE
-    if (where_to_send() == OUTPUT_BLUETOOTH) {
-        bluetooth_send_keyboard(report);
-        return;
-    }
-#endif
```

Comment:
Apparently this change (and other similar changes in this file) removes all usages of `where_to_send()`, and effectively removes the existing support for the `OUTPUT_AUTO` mode.  Where would this logic go after the rework?

Reply:
`OUTPUT_AUTO` is indeterminate when you have a Bluetooth + 2.4GHz capable board. For that reason, it's been aliased to `OUTPUT_NEXT` which will cycle through the different connections instead.

Reply:
That was indeed the intention, but I can see how useful of automatic connection swapping could be.
Keycodes `0.0.6` already have `OUTPUT_AUTO` removed (well, aliased to `QK_OUTPUT_NEXT`) but it's not yet merged to master -- I'll make a separate PR to reshuffle and reintroduce `OUTPUT_AUTO` so that it doesn't result in changes on `master`.